doc/toolbox.1.md

```diff
@@ -40,6 +40,20 @@ access to the user's home directory, the Wayland and X11 sockets, networking
 (including Avahi), removable devices (like USB sticks), systemd journal, SSH
 agent, D-Bus, ulimits, /dev and the udev database, etc..
 
+## Supported distribution images
+
+Toolbox maintains a list of supported distribution images. These images are
+maintained by the team to be compatible with Toolbox. Some commands accept the
+`--distro` and `--release` flags. `--distro` accepts `ID` from `os-release`
+(e.g., on Fedora it is `fedora`). `--release` accepts version number of the
+selected distribution. Each distribution uses sligthly different versioning
+schema and some don't have any at all. Supported distributions are:
```

Comment:
Then:
```
By default, Toolbox tries to create containers for the same operating system
distribution as the host.  If the host is not supported, then it creates a Fedora
container.

However, it's possible to create containers for a different distribution through
the use of the `--distro` and `--release` options that are accepted by the
relevant commands, or their counterparts in the configuration file.  The `--distro`
flag specifies the name of the distribution, and `--release` specifies its release.

Supported host operating systems:
...
...
```

We offer a configuration file now, so we might as well mention it.  :)

Reply:
Reworded the section using suggestions.